marklogic-data-hub-central/ui/src/components/modeling/entity-type-table/entity-type-table.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { render } from '@testing-library/react';
+import userEvent from "@testing-library/user-event";
+import EntityTypeTable from './entity-type-table';
+
+import { getEntityTypes } from '../../../assets/mock-data/modeling';
+
+describe('EntityTypeModal Component', () => {
+  test('Table renders with empty array prop', () => {
+    const { getByText } =  render(
+      <Router>
+        <EntityTypeTable 
+          allEntityTypesData={[]}
+        />
+      </Router>);
+
+    expect(getByText(/No Data/i)).toBeInTheDocument();
+  });
+
+    test('Table renders with mock data', () => {
```

Comment:
We can get rid of corresponding cypress integration test since we have this unit test in place, to reduce duplication and run time.